<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GH6SL2D" failures="1" tests="4" name="PetTest" time="13.568" errors="0" timestamp="2025-03-24T02:20:02 IST">
  <testcase classname="api.test.UserTests2" name="testPostUser" time="5.77"/>
  <testcase classname="api.test.UserTests2" name="testGetUser" time="1.457"/>
  <testcase classname="api.test.UserTests2" name="testUpdateUser" time="2.887"/>
  <testcase classname="api.test.UserTests2" name="testDeleteUser" time="1.521">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests2.testDeleteUser(UserTests2.java:102)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUser -->
</testsuite> <!-- PetTest -->
